engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -29,7 +29,9 @@ object MaterializeWorkflowDescriptorActor {
 
   val RuntimeBackendKey: String = "backend"
 
-  def props(serviceRegistryActor: ActorRef, workflowId: WorkflowId): Props = Props(new MaterializeWorkflowDescriptorActor(serviceRegistryActor, workflowId))
+  def cromwellBackends = CromwellBackends.instance.get
```

Comment:
It might be hard to get the emoji in there but can you put a comment to that effect in here? 

Since complexity and cognitive load are hot topics these days I find the cognitive load of having to figure out why the author determined the raw gets to be safe and/or not worth worrying about to be pretty high